daemon/response.go

```diff
@@ -79,7 +79,7 @@ type respJSON struct {
 	Type       ResponseType `json:"type"`
 	Status     int          `json:"status-code"`
 	StatusText string       `json:"status"`
-	Result     interface{}  `json:"result,omitempty"`
+	Result     interface{}  `json:"result"`
```

Comment:
@zyca can you leave a comment that this field must not be tagged `omitempty` in the future because of backward compatibilty?

Reply:
I thought about that but instead I added a test with that comment.